src/wazuh_modules/syscollector/syscollector_common.c

```diff
@@ -0,0 +1,322 @@
+/*
```

Comment:
Please remove this file. It was deleted in another development.

Reply:
It was removed in this commit fcff159439fdcee38c8870c942baacb153c0954f